constants.go

```diff
@@ -57,6 +56,8 @@ const (
     <rng model='virtio'>
       <backend model='random'>/dev/urandom</backend>
     </rng>
+    <memballoon model='none'>
+    </memballoon>
```

Comment:
As per https://libvirt.org/formatdomain.html#memory-balloon-device shouldn't this will be something like?
```xml
<devices>
  <memballoon model='none'/>
</devices>
```

Reply:
It's the same, just different annotation. Changed.

Reply:
> It will be automatically added when appropriate, so there is no need to explicitly add this element in the guest XML 

It should even be removed it seems

Reply:
> > It will be automatically added when appropriate, so there is no need to explicitly add this element in the guest XML
> 
> It should even be removed it seems

It's added by default, always has been:
```
    <memballoon model="virtio">
      <address type="pci" domain="0x0000" bus="0x00" slot="0x06" function="0x0"/>
    </memballoon>
```

Reply:
This is my point, if it's added by default, we don't need to add it ourselves

Reply:
I'm not adding it, I'm removing it (model='none').

Reply:
aaah ok, it needs to be explicitly disabled, thanks for the explanation :)